tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -93,6 +93,7 @@ options:
 	rootfs-initrd
 	shim-v2
 	tdvf
+	ovmf
```

Comment:
@fitzthum, we're trying to keep those alphabetically sorted. Mind to move this entry after nydus-tarball and before qemu-tarball ?